plugins/woocommerce/src/Internal/FraudProtection/FraudProtectionController.php

```diff
@@ -58,10 +68,80 @@ public function on_init(): void {
 			return;
 		}
 
+		// Check Jetpack connection status and log if not connected.
+		$this->check_jetpack_connection();
+
 		// Future implementation: Register hooks and initialize components here.
 		// For now, this is a placeholder for the infrastructure.
 	}
 
+	/**
+	 * Check Jetpack connection status and log warning if not connected.
+	 *
+	 * This implements the fail-open pattern: the feature will continue to work
+	 * even if Jetpack is not connected, but API calls will fail gracefully and
+	 * allow all sessions through.
+	 *
+	 * @return void
+	 */
+	private function check_jetpack_connection(): void {
+		$connection_status = $this->connection_manager->get_connection_status();
+
+		if ( ! $connection_status['connected'] ) {
+			self::log(
+				'warning',
+				sprintf(
+					'Fraud Protection is enabled but Jetpack connection is not available: %s (Error: %s). Fraud protection will fail open and allow all sessions.',
+					$connection_status['error'],
+					$connection_status['error_code']
+				)
+			);
+		}
+	}
+
+	/**
+	 * Display admin notice when Jetpack connection is not available.
+	 *
+	 * @internal
+	 */
+	public function on_admin_notices(): void {
+		// Only show if feature is enabled.
+		if ( ! $this->feature_is_enabled() ) {
+			return;
+		}
+
+		// Only show on WooCommerce settings page.
+		$screen = get_current_screen();
+		if ( ! $screen || 'woocommerce_page_wc-settings' !== $screen->id ) {
+			return;
+		}
+
+		$connection_status = $this->connection_manager->get_connection_status();
+		if ( $connection_status['connected'] ) {
+			return;
+		}
+
+		$settings_url = admin_url( 'admin.php?page=wc-settings&tab=advanced&section=features' );
+
+		?>
+		<div class="notice notice-warning is-dismissible">
+			<p>
+				<strong><?php esc_html_e( 'Fraud Protection Warning:', 'woocommerce' ); ?></strong>
```

Comment:
Nit: Should we use sentence case here? This might apply to some other UI elements in this PR.

Reply:
Fixed with 3185042773b0b56952e98353e38fd0389d660a8c